theme/extras/_blocks.scss

```diff
@@ -17,9 +17,4 @@
       }
     }
   }
-
-  > * {
-    padding-right: 24px !important;
```

Comment:
questa modifica mi fa paurissima

Reply:
Questa modifica dei px così infatti non c'è più, era la prima proposta di soluzione che poi è stata rivista in maniera totalmente differente se vedi le modifiche della PR allo stato attuale.